python/lsst/ts/observatory/control/maintel/mtcs.py

```diff
@@ -1695,6 +1745,80 @@ async def _ready_to_take_data(self) -> None:
         """Placeholder, still needs to be implemented."""
         # TODO: Finish implementation.
 
+    async def open_m1m3_booster_valve(self) -> None:
+        """Open M1M3 booster valves."""
+        if self.check.mtm1m3:
+            await self._handle_m1m3_booster_valve(open=True)
+        else:
+            self.log.info("M1M3 check disabled.")
+
+    async def close_m1m3_booster_valve(self) -> None:
+        """Close M1M3 booster valves."""
+        if self.check.mtm1m3:
+            await self._handle_m1m3_booster_valve(open=False)
+
+    async def _handle_m1m3_booster_valve(self, open: bool) -> None:
+        """Handle opening the M1M3 booster valves"""
+
+        desired_state = "open" if open else "close"
```

Comment:
For almost all uses of this variable (except lines 1776 and 1802), the value should be `closed` instead of `close`.